src/settings/index.ts

```diff
@@ -679,3 +679,5 @@ export const VENUES_WITH_CHAT_REQUIRED = [
 export const VENUE_CHAT_MESSAGES_COUNTER_SHARDS_COUNT = 10;
 
 export const NON_EXISTENT_FIRESTORE_ID = "NON_EXISTENT_FIRESTORE_ID";
+
+export const INVALID_SLUG_CHARS_REGEX = /[^a-zA-Z0-9]/g;
```

Comment:
This should be refactored so that we only define this constant once.

Reply:
Interested. Please do provide me with a working POC. https://github.com/sparkletown/sparkle/pull/2509#discussion_r740245059